docs/dev-guide/building.md

```
@@ -0,0 +1,31 @@
+## Install the dependencies
+
+```shell
+$ npm install
+```
+
+## Running in development
+
+```shell
+$ npm start
+```
+The application should now be served in debug mode at http://localhost:3010. 
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)